pages/project-registry/profiles.mdx

```diff
@@ -11,3 +15,32 @@ struct Profile {
     address anchor;
 }
 ```
+| Field | Data Type | Notes |
+| --- | --- | --- |
+| nonce | uint256 |  | 
+| name | string | The profile name. | 
+| metadata | Metadata | See the metadata section below for more details.|
+| owner | address | The wallet address of the profile owner. |
+| anchor | address | The unique id assigned to the profile when it is created. |
+
+### Metadata
+Offchain metadata can be stored in the project registry using the Metadata 
+struct.
+```solidity
+struct Metadata {
+    uint256 protocol;
+    string pointer;
+}
+```
+| Field | Data Type | Notes |
+| --- | --- | --- |
+| protocol | uint256 | An id from the list of available protocols. | 
+| pointer | string | The pointer to the off chain data. | 
+
+Currently, the only protocol id available is '1' for IPFS. For more details on
+details see [`MetaPtrProtocol.sol`](https://github.com/allo-protocol/allo-contracts/blob/main/docs/MetaPtrProtocol.md)
+
+## Creating and Updating Profiles
```

Comment:
Instead of this heading, can we have a short paragraph that says "read more" and links to the Integrating with the Project Registry page (#36 )